validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/BlobsetSequences.java

```diff
@@ -46,4 +64,26 @@ public void endpoint_config_connection_error() {
     });
   }
 
+  @Test
+  @Description("Push endpoint config message to device that results in success.")
+  public void endpoint_config_connection_success() {
+    BlobBlobsetConfig config = new BlobBlobsetConfig();
+    config.phase = BlobPhase.FINAL;
+    config.base64 = endpointConfigBase64PayloadHostname(ENDPOINT_CONFIG_HOSTNAME);
```

Comment:
method name seems off -- should the "Hostname" be there?

Reply:
Right, for a while I had a different function that generated a payload with client_id. But now, it's removed. So I also renamed the function.